gprofiler/main.py

```diff
@@ -391,10 +391,11 @@ def parse_cmd_args():
     )
 
     parser.add_argument(
-        "--log-cpu-usage",
+        "--log-usage",
         action="store_true",
         default=False,
-        help="Log CPU usage (per cgroup) on each profiling iteration. Works only when gProfiler runs as a container",
+        help="Log CPU & memory usage (per my cgroup) on each profiling iteration."
```

Comment:
As we discussed in the previous PR (CPU logging), this isn't meant for users but for devs of gProfiler.

But I agree it could be more clear - will update